src/3d/qgs3daxis.cpp

```diff
@@ -603,21 +554,22 @@ void Qgs3DAxis::createMenu()
   mMenu->addMenu( vertPosMenu );
 
   // axis view menu
-  QAction *viewHomeAct = new QAction( tr( "&Home" ) + "\t Ctrl+1", mMenu );
-  QAction *viewTopAct = new QAction( tr( "&Top" ) + "\t Ctrl+5", mMenu );
+  // Make sure to sync the key combinations with QgsCameraController::keyboardEventFilter()!
+  QAction *viewHomeAct = new QAction( tr( "&Home" ) + "\t Ctrl+5", mMenu );
+  QAction *viewTopAct = new QAction( tr( "&Top" ) + "\t Ctrl+9", mMenu );
   QAction *viewNorthAct = new QAction( tr( "&North" ) + "\t Ctrl+8", mMenu );
   QAction *viewEastAct = new QAction( tr( "&East" ) + "\t Ctrl+6", mMenu );
   QAction *viewSouthAct = new QAction( tr( "&South" ) + "\t Ctrl+2", mMenu );
   QAction *viewWestAct = new QAction( tr( "&West" ) + "\t Ctrl+4", mMenu );
-  QAction *viewBottomAct = new QAction( tr( "&Bottom" ), mMenu );
+  QAction *viewBottomAct = new QAction( tr( "&Bottom" ) + "\t Ctrl+3", mMenu );
 
-  connect( viewHomeAct, &QAction::triggered, this, &Qgs3DAxis::onCameraViewChangeHome );
-  connect( viewTopAct, &QAction::triggered, this, &Qgs3DAxis::onCameraViewChangeTop );
-  connect( viewNorthAct, &QAction::triggered, this, &Qgs3DAxis::onCameraViewChangeNorth );
-  connect( viewEastAct, &QAction::triggered, this, &Qgs3DAxis::onCameraViewChangeEast );
-  connect( viewSouthAct, &QAction::triggered, this, &Qgs3DAxis::onCameraViewChangeSouth );
-  connect( viewWestAct, &QAction::triggered, this, &Qgs3DAxis::onCameraViewChangeWest );
-  connect( viewBottomAct, &QAction::triggered, this, &Qgs3DAxis::onCameraViewChangeBottom );
+  connect( viewHomeAct, &QAction::triggered, this, [this]() { mCameraController->rotateCameraToHome(); } );
+  connect( viewTopAct, &QAction::triggered, this, [this]() { mCameraController->rotateCameraToTop(); } );
+  connect( viewNorthAct, &QAction::triggered, this, [this]() { mCameraController->rotateCameraToNorth(); } );
+  connect( viewEastAct, &QAction::triggered, this, [this]() { mCameraController->rotateCameraToEast(); } );
+  connect( viewSouthAct, &QAction::triggered, this, [this]() { mCameraController->rotateCameraToSouth(); } );
+  connect( viewWestAct, &QAction::triggered, this, [this]() { mCameraController->rotateCameraToWest(); } );
+  connect( viewBottomAct, &QAction::triggered, this, [this]() { mCameraController->rotateCameraToBottom(); } );
```

Comment:
You can skip the lambda here, it's not required:

```suggestion
  connect( viewHomeAct, &QAction::triggered, mCameraController, &QgsCameraController::rotateCameraToHome );
  ...etc...
```

Reply:
Thanks, done.